api/config/routes.rb

```diff
@@ -166,6 +166,11 @@
 
         resources :menus, only: %i[index show]
         resources :cms_pages, only: %i[index show]
+
+        resources :wishlists do
+          get 'default', on: :collection
+          resources :wished_products, only: [:create, :update, :destroy]
```

Comment:
Hi @damianlegawiec 

Quick question, do you want `:wished_products` to be nested inside `:wishlists` as they were in the extension, or each a stand alone resource?

Reply:
@MatthewKennedy let's not follow the extension design here (which is not optimal), let's add

`POST /wishlist/<ID>/add_item` with `variant_id` and `quantity` parama - similar to Cart Add Item endpoint
and
`DELETE /wishlist/<ID>/remove_item/<wished_product.id>` - similar to https://api.spreecommerce.org/docs/api-v2/b3A6MTg1MTUyNTg-remove-line-item

Reply:
Will do that.

Reply:
Hi @damianlegawiec 

As I work on this I'm getting the sense also looking at other websites wish lists, the `remark` column is not really needed, what do you think?

It seems to be the only column standing between a nice clean `set_item_quantity` action in the wishlist controller Vs. having to have an separate update action for `wished_variant`

If we dropped the remark column, we could have these actions on the wishlists_controller: `add_item`, `remove_item`, `set_item_quantity`.

Any thoughts on this?

Reply:
@MatthewKennedy let's drop the remark, new MetaData feature could resolve this use case

Reply:
@damianlegawiec Awesome, thats what I was thinking.

Reply:
@damianlegawiec 

Do you want those actions `add_item`, `remove_item`, `set_item_quantity` moving out to service objects or just leaving in the controller for now?